core/src/main/scala/slamdata/algebras/GitData.scala

```diff
@@ -0,0 +1,21 @@
+package precog.algebras
+
+import github4s.GithubResponses.GHResponse
+import github4s.domain.Ref
+
+trait GitData[F[_]] extends github4s.algebras.GitData[F] {
```

Comment:
Trivia note! This is the very first time I've ever worked with or reviewed code that is *actually* using a finally tagless encoding.